python/dgl/sampling/neighbor.py

```diff
@@ -14,9 +14,44 @@
     'sample_neighbors_biased',
     'select_topk']
 
-def sample_etype_neighbors(g, nodes, etype_field, fanout, edge_dir='in', prob=None,
-                           replace=False, copy_ndata=True, copy_edata=True, etype_sorted=False,
-                           _dist_training=False, output_device=None):
+def _prepare_edge_arrays(g, arg):
```

Comment:
As @jermainewang suggested earlier, "briefly doc what this internal function is for"